lib/Agama/agama_base.pm

```diff
@@ -0,0 +1,20 @@
+## Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+# Summary: First installation using D-Installer current CLI (only for development purpose)
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+package Agama::agama_base;
+use base 'y2_base';
```

Comment:
could we use `opensusebasetest`? even in y2_base there are plenty of sle specific stuff.

Reply:
does not work with opensusebasetest, not sure why http://waaa-amazing.suse.cz/tests/18326